### PR TITLE
Add Southern company Integration

### PR DESCRIPTION
**Pull request by Lash-L**
## Proposed change

Southern Company is a energy company which owns 3 major power companies in the southeast, one in Georgia, Alabama, and Mississippi. this integration aims to allow customers in those regions to get their energy data into Home Assistant.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/87748
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4115
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/southern_company.markdown
Comment (fabaff) [inline source/_integrations/southern_company.markdown]:
```suggestion
The Southern Company sensor platform allows you to get energy information from [Southern Company Energy Companies](https://www.southerncompany.com/about/our-companies.html).
```

File: source/_integrations/southern_company.markdown
Comment (fabaff) [inline source/_integrations/southern_company.markdown]:
```suggestion
To add this platform to your installation, You will need your Southern Company login email and password.
```

File: source/_integrations/southern_company.markdown
Comment (fabaff) [inline source/_integrations/southern_company.markdown]:
```suggestion
- Lower Projected monthly Usage: The lower bounds of how much energy Southern Company projects you will use
```

File: source/_integrations/southern_company.markdown
Comment (fabaff) [inline source/_integrations/southern_company.markdown]:
```suggestion
- Higher projected monthly usage: The upper bounds of how much energy Southern Company projects you will use
```

File: source/_integrations/southern_company.markdown
Comment (fabaff) [inline source/_integrations/southern_company.markdown]:
```suggestion
- Lower projected monthly cost: The lower bounds of how much energy Southern Company projects your bill will cost
```

**Comment by Lash-L** [comment]
Thanks @fabaff!

File: source/_integrations/southern_company.markdown
Comment (frenck) [inline source/_integrations/southern_company.markdown]:
```suggestion
The Southern Company integration allows you to get energy information from [Southern Company Energy Companies](https://www.southerncompany.com/about/our-companies.html).
```

File: source/_integrations/southern_company.markdown
Comment (frenck) [inline source/_integrations/southern_company.markdown]:
```suggestion
To add this integration to your installation, You will need your Southern Company login email and password.
```

File: source/_integrations/southern_company.markdown
Comment (frenck) [inline source/_integrations/southern_company.markdown]:
```suggestion
- Monthly cost: The most recent update of your current bill total.
- Monthly consumption: The most recent update of your current energy usage.
- Average daily usage: How much energy you have used on average in this billing cycle.
- Average daily cost: How much it costs on average to run your electricity in this billing cycle.
- Lower Projected monthly Usage: The lower bounds of how much energy Southern Company projects you will use.
- Higher projected monthly usage: The upper bounds of how much energy Southern Company projects you will use.
- Lower projected monthly cost: The lower bounds of how much energy Southern Company projects your bill will cost.
- Higher projected monthly cost: The upper bounds of how much energy southern company projects your bill will cost.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by Lash-L** [comment]
> ✅ Approved. Can be merged as soon as the parent PR gets merged. ⚠️ The `ha_release` might need an update before merging this one in.

Thanks @frenck, I'll keep the ha_release in mind when the parent PR is approved

File: source/_integrations/southern_company.markdown
Comment (tronikos) [inline source/_integrations/southern_company.markdown:48]: Can you add delayed data in the energy dashboard? https://community.home-assistant.io/t/import-historic-sensor-data/340980 suggests you cannot although things might have changed since Sep '21.

Comment (Lash-L) [reply source/_integrations/southern_company.markdown:48]: Yes. I copy what tibber, an already implemented integration does

Comment (frenck) [reply source/_integrations/southern_company.markdown:48]: > Can you add delayed data in the energy dashboard?

This is not related to the contents of this PR.

Comment (tronikos) [reply source/_integrations/southern_company.markdown:48]: Sorry my question wasn't a feature request. It was a clarification question if what @Lash-L is doing actually works. So the energy dashboard keeps getting updated and shows a gap for the past 2 days? Or is the data time shifted by 2 days? I'm asking because my energy provider also provides data delayed by 2 days and https://community.home-assistant.io/t/pg-e-energy-monitoring-integration/336038/38 suggested it wouldn't be possible to import that data in the dashboard.

Comment (Lash-L) [reply source/_integrations/southern_company.markdown:48]: @tronikos message me on the home assistant community forum. My username is the same as here Lash-L. This isn't the place to talk about this as every time a response is made fabaff and Frenck are getting notified

File: source/_integrations/southern_company.markdown
Comment (tronikos) [inline source/_integrations/southern_company.markdown:31]: s/You/you/

Comment (tronikos) [reply source/_integrations/southern_company.markdown:31]: And is it email or username?

**Comment by Lash-L** [comment]
To be honest - I barely have enough time for the integrations I have right now, I don't need to add a third that I am responsible for. I'm going to close this issue - and if anyone comes upon it in the future, feel free to take the work I have done and I would be happy to help you get it up and running, but I don't want to be the primary code owner

If anyone wants to use this integration - you can use the HACs version of the integration: https://github.com/Lash-L/southern-company-hacs

**Pull request closed without merging**
